src/components/nav.jsx

```diff
@@ -0,0 +1,42 @@
+import React from "react";
+import { Link } from "react-router-dom";
+
+function Nav() {
+  return (
+    <nav>
+      <div className="nav">
+        <div className="first-nav">
+          <img
+            src="https://res.cloudinary.com/de5it79dw/image/upload/v1650394802/pixelvideologo_bbmywr.png"
+            alt=""
+          />
+        </div>
+        <div className="mid-nav flex-row">
+          <ul className="nav-links">
+            <Link className="li-link-a" to="/">
+              {" "}
+              <li className="li-link">Explore</li>
+            </Link>
+            <Link className="li-link-a" to="/product">
+              {" "}
+              <li className="li-link">Playlists</li>
+            </Link>
```

Comment:
- Instead of `<Link />`, you can use `<NavLink />` so that you can **highlight the active link easily.**